.github/workflows/push.yml

```diff
@@ -10,11 +10,6 @@ jobs:
       - uses: actions/checkout@v2
       - name: docker build
         run: docker build -t slimm609/checksec:latest .
-      - name: Login to DockerHub
```

Comment:
why is this being removed?

Reply:
Apologies, I don't mean to touch the push.yml or related files... with the GitHub workflow failing I was trying out something. Pl ignore it.
The first part 'minor: update path to kernel Documentation/.../sysctl.rst file' is the relevant patch only.

Reply:
Please rebase and remove that part. I can’t merge 1/2 of a merge request